databricks/koalas/tests/test_series.py

```diff
@@ -232,6 +232,33 @@ def test_drop_duplicates(self):
         self.assert_eq(kser.sort_index(), pser.sort_index())
         self.assert_eq(kdf, pdf)
 
+    def test_reindex(self):
+        index = ["A", "B", "C", "D", "E"]
+        pser = pd.Series([1.0, 2.0, 3.0, 4.0, None], index=index, name="x")
+        kser = ks.from_pandas(pser)
+
+        self.assert_eq(pser, kser, almost=True)
+
+        self.assert_eq(
+            pser.reindex(["A", "B"]).sort_index(),
+            kser.reindex(["A", "B"]).sort_index(),
+            almost=True,
```

Comment:
I think we can remove `almost` now since #1722   😸

Reply:
Thanks I did not notice the change !